modules/irsa/main.tf

```diff
@@ -1,3 +1,7 @@
+locals {
+  eks_oidc_issuer_url = "${replace(var.eks_oidc_provider_arn, "/^(.*provider/)/", "")}:sub"
```

Comment:
@vara-bonthu should this have `:sub` ?

Reply:
This is breaking things, I believe. Look at https://github.com/aws-ia/terraform-aws-eks-blueprints/issues/837